content/connaitre-des-applications-a-videoprojeter-pour-animer-et-gerer-sa-classe.json

```
@@ -0,0 +1,8 @@
+{
+  "title": "Connaître des applications à vidéoprojeter pour animer et gérer sa classe",
+  "description": "Tirage au sort, chronomètre, QR code, fichier audio : beaucoup d'outils existent de manière indépendante. Certaines applications centralisent toutes ses fonctionnalités pour permettre d'animer et de gérer des activités et les différents moments d'un cours. Les détails dans cette vidéo !",
```

Le `et … et` est compliqué à lire, il faut peut-être revoir la tournure ? 🤔  
```suggestion
  "description": "Tirage au sort, chronomètre, QR code, fichier audio : beaucoup d'outils existent de manière indépendante. Certaines applications centralisent toutes ses fonctionnalités pour permettre d'animer, de gérer des activités et les différents moments d'un cours. Les détails dans cette vidéo !",
```